lib/pay/paddle_billing/billable.rb

```diff
@@ -33,7 +33,13 @@ def customer
           sc
         end
       rescue ::Paddle::Error => e
-        raise Pay::PaddleBilling::Error, e
+        sc = Paddle::Customer.list(email:).data.first
```

Comment:
Paddle's CustomerAlreadyExists error message contains the ID in it, so we can parse it out of the string I think:

> "customer email conflicts with customer of id %s"

Reply:
Can always do that as a fallback if it can't find the value. 👍